### ISSUE TITLE
Week 4 review ticket

### ISSUE THREAD

**Issue by EliGilmour** [EliGilmour/Eli-1#6]
[Psychology notes](https://eligilmour.github.io/Eli-1/fastpages/markdown/2022/09/20/psychology.html)
[Collaboration notes](https://eligilmour.github.io/Eli-1/fastpages/markdown/2022/09/20/collabrationnotes1.html)
[Video](https://youtu.be/Vit0CqrO_70)

**Comment by GeneChang1** [comment]
Don't push changes to show local changes, hurt presentation. Edit out errors, and try fixing them. Didn't need notes in the video.

**Comment by Nathan-Capule** [comment]
Did not cut out the error showing minimal editing. Was able to show local change and was within the timeframe.

**Comment by jm1021** [comment]
Start was a little abrupt.  Commit step was good, but not showing difference between local and github server.    Not sure you covered all requirements. 2.5

**Comment by drewreed2005** [comment]
Shows Python server running with edit, but Fastpages requirement has not been completed. Good job incorporating collaboration and Focus and Habits content, though. 2.4

**Comment by trentcardall** [comment]
2.5 Showed flask server running but did not show fastpages server and could have used more editing

**Comment by DreamerBlock** [comment]
hard to follow didn't show fast pages 2.4

**Comment by TheGerbil21** [comment]
The video needed some cuts and better audio quality. Very difficult to follow despite the changes being shown for the most part. 2.5

**Comment by CubNavarro** [comment]
the video needed edits and didn't meet all of the required material 2.4

**Comment by jagermi3ster** [comment]
2.5 showed flask didn't have fastpages or edits
